src/runtime/components/forms/Range.vue

```diff
@@ -92,7 +92,8 @@ export default defineComponent({
   setup (props, { emit }) {
     const { ui, attrs, attrsClass } = useUI('range', props.ui, config)
 
-    const { emitFormChange, inputId, color, size, name } = useFormGroup(props, config)
+    const { emitFormChange, inputId, color, size: formGroupSize, name } = useFormGroup(props, config)
+    const size = computed(() => ui.value.size[formGroupSize] ? formGroupSize : config.default?.size)
```

Comment:
```suggestion
    const size = computed(() => ui.value.size[formGroupSize] ? formGroupSize : (props.size ?? config.default?.size))
``` 
How about this ? I'm not sure if `range` component still need to custom in `formGroup` thou.